english/super-powers/practice-flying.md

```diff
@@ -0,0 +1,5 @@
+You can't save the world until you harness your new found power. You jump up and down a few times but it doesn't amount to much. You need to get higher. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.